docker-compose.yml

```diff
@@ -1,19 +1,21 @@
 version: "3.9"
 services:
   db:
-    image: postgres:latest
+    image: postgres:16
     restart: always
     ports:
       - "5432:5432"
     volumes:
-      - ./tmp/db:/var/lib/postgresql/data
+      - ./tmp/db:/var/lib/postgresql/data:delegated
+      - ./tmp/shared:/tmp/shared:cached
     environment:
       - POSTGRES_USER=admin
       - POSTGRES_PASSWORD=password
       - POSTGRES_DB=commitchange_development_legacy
   web:
     build: .
     volumes:
+      - ${HOME}${USERPROFILE}/.netrc:/home/app/.netrc # for heroku credentials
```

Comment:
Just so you know: `USERPROFILE` isn't always set on Linux.

Reply:
Yeah,  as someone who has never run linux, I'm probably the wrong person to streamline the linux dev experience.